prison-spigot/src/main/java/tech/mcprison/prison/spigot/placeholder/PlaceHolderAPIIntegration.java

```diff
@@ -0,0 +1,82 @@
+package tech.mcprison.prison.spigot.placeholder;
+
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import tech.mcprison.prison.integration.PlaceholderIntegration;
+import tech.mcprison.prison.internal.Player;
+import tech.mcprison.prison.ranks.PrisonRanks;
+import tech.mcprison.prison.ranks.data.Rank;
+import tech.mcprison.prison.ranks.data.RankLadder;
+import tech.mcprison.prison.ranks.data.RankPlayer;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Function;
+
+public class PlaceHolderAPIIntegration extends PlaceholderExpansion implements PlaceholderIntegration {
+
+	private boolean pluginInstalled;
+
+	public PlaceHolderAPIIntegration() {
+		pluginInstalled = Bukkit.getPluginManager().isPluginEnabled("PlaceholderAPI");
+	}
+
+	@Override
+	public void registerPlaceholder(String placeholder, Function<Player, String> action) {
+		this.register();
+	}
+
+	@Override
+	public String getProviderName() {
+		return "PlaceHolderAPI";
+	}
+
+	@Override
+	public boolean hasIntegrated() {
+		return pluginInstalled;
+	}
+
+	@Override
+	public String getAuthor(){
+		return "PrisonTeam";
+	}
+
+	@Override
+	public String getIdentifier(){
+		return "Prison";
+	}
+
+	@Override
+	public String getVersion(){
+		return "1.0.0";
```

Comment:
Could we maybe hook this up to the one in plugin.yml via SpigotPrison? Is possible, but haven't done it in a while so it's worth googling.